features/testdebt_generic_test_data_step.feature

```diff
@@ -0,0 +1,28 @@
+@deed_overrides @wip
+Feature: Override deed variables
+
+Acceptance Criteria
+- Can override ALL deed variables from one step
+
+@techdebt @techdebt1
+Scenario: WIP for default deed
+Given I create default deed with <2> borrowers
+And borrower <2> views the deed
+And borrower <1> views the deed
+
+@techdebt @techdebt2
+Scenario: WIP for override
+Given I create default deed with <2> borrowers
+And I override deed with "forename:TIMMY:2"
+
+@techdebt @techdebt3
+Scenario:  WIP create deed with override
+Given I create default deed with <3> borrowers,"forename:TIMMY:2"
```

Comment:
Really like this one step for setting up the deed, though as discussed not so sure about the codey bit for the optional parameters at the end. I wonder if there is a way we can have these optional parameters in a more readable format without sacrificing the compactness.